inbc-program/README.md

```diff
@@ -404,6 +407,54 @@ inbc query --option hw
 inbc query --option sw
 ```
 
+## SOURCE APPLICATION UPDATE
+### Description
+Updates Application sources that are used to update the system
+NOTE: Currently this only works on Ubuntu
+
+### Usage
+```
+inbc source application update {-fileName, --f=FILEPATH} {-sources, --s=SOURCES}
+```
+
+### Examples
+#### Update an application source file
+```
+sudo inbc source application update --f /etc/apt/sources.list.d/google-chrome.list --s "deb [arch=amd64] https://dl.google.com/linux/chrome/deb/ stable test" "debsrc [arch=amd64] https://dl.google.com/linux/chrome/deb/ stable test2
```

Comment:
filename would just be "google-chrome.list" for list and remove.. I think same convention should be followed for update and add.